lblod/frontend-embeddable-notule-editor
# Refactor logic for showing UI sections based on active plugins

## Overview
Moves the single insert menu to multiple sections, similar to the [most recent designs](https://www.figma.com/design/tfBSv7999dzz1sy6svMfsY/GN_editor), though without the actual style changes.

Makes handling sections which show for one of a list of plugins much simpler to manage. Also removes the UI dependency of lpdc plugin to besluit plugin.

##### connected issues and PRs:
Part of: https://binnenland.atlassian.net/browse/GN-5164

### Setup
N/A

### How to test/reproduce
You should see for the sample pages with plugins that the insert menu is split into sections. Removing the 'besluit' plugin but leaving the 'lpdc' plugin means that the insert lpdc button still appears in the sidebar.

### Challenges/uncertainties
N/A

### Checks PR readiness
